package.json

```diff
@@ -19,14 +19,14 @@
   "module": "dist/kore-web-sdk.esm.browser.js",
   "repository": {
     "type": "git",
-    "url": "git@github.com:Koredotcom/web-kore-sdk.git#v2/9.3.0"
+    "url": "git@github.com:KambagiriswamyBo-Kore/web-kore-sdk.git#v2/9.3.0"
```

Comment:
@KambagiriswamyBo-Kore 

should point to koredotcom repo

Reply:
@RajasekharBa-Kore Updated the github url